src/openrct2/windows/EditorObjectSelection.cpp

```diff
@@ -825,7 +825,7 @@ static void window_editor_object_selection_mouseup(rct_window *w, rct_widgetinde
     }
     case WIDX_FILTER_STRING_BUTTON:
         //window_text_input_open(w, widgetIndex, STR_OBJECT_SEARCH, STR_OBJECT_SEARCH_DESC, STR_STRING, (uint32)_filter_string, 40);
-        window_start_textbox(w, widgetIndex, STR_STRING, _filter_string, 40);
```

Comment:
Any idea why this was 40 while the definition said 41? Will it not cause problems replacing both by "255"?

Reply:
I think the original intent was to make sure it couldn't overwrite the null-terminator, but this doesn't seem to be neccesary. With a value of 40, only 39 characters can be entered in the search box (and not because `safe_strcpy` takes care of it)